src/components/common/EditableWordCloudDataCard.js

```diff
@@ -207,27 +207,23 @@ class EditableWordCloudDataCard extends React.Component {
           <DownloadButton />
         </ListItemIcon>
       </MenuItem>,
-      !onDownload && (
-        <MenuItem
-          className="action-icon-menu-item"
-          disabled={this.state.editing} // can't download until done editing
-          onClick={() => this.downloadCsv(2)}
-        >
-          <ListItemText><FormattedMessage {...localMessages.downloadBigramCSV} /></ListItemText>
-          <ListItemIcon>
-            <DownloadButton />
-          </ListItemIcon>
-        </MenuItem>
-      ),
-      !onDownload && (
-        <MenuItem
-          className="action-icon-menu-item"
-          disabled={this.state.editing} // can't download until done editing
-          onClick={() => this.downloadCsv(3)}
-        >
-          <FormattedMessage {...localMessages.downloadTrigramCSV} />
-        </MenuItem>
-      ),
+      <MenuItem
+        className="action-icon-menu-item"
+        disabled={this.state.editing} // can't download until done editing
+        onClick={() => this.downloadCsv(2)}
+      >
+        <ListItemText><FormattedMessage {...localMessages.downloadBigramCSV} /></ListItemText>
+        <ListItemIcon>
+          <DownloadButton />
+        </ListItemIcon>
+      </MenuItem>,
+      <MenuItem
+        className="action-icon-menu-item"
+        disabled={this.state.editing} // can't download until done editing
+        onClick={() => this.downloadCsv(3)}
+      >
+        <FormattedMessage {...localMessages.downloadTrigramCSV} />
```

Comment:
Do we want a download icon here too?

![Screen Shot 2020-06-17 at 2 27 30 PM](https://user-images.githubusercontent.com/5265058/84935978-8c69ea80-b0a7-11ea-9ebc-77f82f4a0dbb.png)